internal/app/machined/internal/server/v1alpha1/v1alpha1_server.go

```diff
@@ -1228,19 +1227,19 @@ func (s *Server) Logs(req *machine.LogsRequest, l machine.MachineService_LogsSer
 
 	switch {
 	case req.Namespace == constants.SystemContainerdNamespace || req.Id == "kubelet":
-		var options []runtime.LogOption
+		var opts []runtime.LogOption
```

Comment:
can we please skip in the future changing every line for unrelated changes, and close to beta release specifically. it makes more work for reviewers.

Reply:
Dropped all irrelevant stuff.

Reply:
Sorry! I thought you wanted me to drop this unrelated stuff.